.circleci/config.yml

```diff
@@ -1135,207 +1079,3 @@ workflows:
           requires:
             - assets_precompile
           <<: *only-master-filter
-
-
-######## Nightly workflows
```

Comment:
Why do we remove the nightly's?

It is good to keep them, at least for postgres and oracle because the other workflows are manual approval only

Reply:
This is what I mentioned yesterday that I don't know how to do... As I said, the goal is to run the tests once for each DB + JS, but I don't know how to do that and I suspected with this change they wouldn't run at all.
I don't want them to run twice, that is why I tried by removing those lines, but I felt it was wrong.
I need help though :stuck_out_tongue:

Reply:
This :smile: 
![image](https://user-images.githubusercontent.com/11318903/68762179-f6a04580-0615-11ea-958d-e4d519983070.png)

Reply:
Just keep them, the executor should be the same as for the Pull Requests workflow

Validate the config before pushing with: `circleci config validate`

Reply:
Awesome! Thanks! :muscle: :dancer: